content/en/blog/2024/otel-errors/index.md

```diff
@@ -0,0 +1,286 @@
+---
+title: Dude, Where's My Error? How OpenTelemetry Records Errors
```

Comment:
```suggestion
title: Dude, where's my error? How OpenTelemetry records errors
```

I know that titles for KubeCon are in captial case, but I think we have that somewhere in some style gudies to not do with our titles